lib/synced/strategies/full.rb

```diff
@@ -241,7 +242,7 @@ def search_param_value_for_lambda(func)
       end
 
       def resource_name
-        @model_class.to_s.tableize
+        @synced_endpoint.to_s.tableize
```

Comment:
not sure if we should tableize all endpoints, I'd expect that when we pass something explicitly we don't want to have it processed afterwards. But let's keep it like that for now.